util/godot/core/mouse_button.h

```diff
@@ -14,7 +14,7 @@
 using namespace godot;
 
 #define ZN_GODOT_MouseButton_NONE MouseButton::MOUSE_BUTTON_NONE
-#define ZN_GODOT_MouseButton_MASK_MIDDLE MouseButton::MOUSE_BUTTON_MASK_MIDDLE
+#define ZN_GODOT_MouseButton_MASK_MIDDLE MouseButtonMask::MOUSE_BUTTON_MASK_MIDDLE
```

Comment:
If GDExtension still requires using masks, the code that checks the mouse middle button would have to change, which further complicates this abstraction...

Reply:
BitField seems to be already in GDExtension, it's in the API JSON.